{
 "pci_projects_project_instances_instance_attach-private-network_title": "Ein oder mehrere private Netzwerke mit Ihrer Instanz verbinden",
 "pci_projects_project_instances_instance_attach-private-network_submit_label": "Verbinden",
 "pci_projects_project_instances_instance_attach-private-network_cancel_label": "Abbrechen",
 "pci_projects_project_instances_instance_attach-private-network_network_placeholder": "Suchen",
 "pci_projects_project_instances_instance_attach-private-network_error_no_compatible_network": "Sie verfügen über kein kompatibles privates Netzwerk, mit dem Sie diese Instanz verbinden können.",
 "pci_projects_project_instances_instance_attach-private-network_success_message": "Das private Netzwerk {{ network }} wurde mit der Instanz {{ instance }} verbunden.",
 "pci_projects_project_instances_instance_attach-private-network_success_multiple_message": "Die privaten Netzwerke wurden mit der Instanz {{ instance }} verbunden.",
 "pci_projects_project_instances_instance_attach-private-network_error_attach": "Bei der Aktualisierung der Instanz ist ein Fehler aufgetreten: {{ message }}"
}